tests/oauth2/rfc6749/endpoints/test_error_responses.py

```diff
@@ -237,7 +237,6 @@ def test_unauthorized_client(self):
 
     def test_access_denied(self):
         self.validator.authenticate_client.side_effect = self.set_client
-        self.validator.confirm_redirect_uri.return_value = False
```

Comment:
In the new flow, `confirm_redirect_uri` is not called because no `redirect_uri` is present in the request and `get_default_redirect_uri` return None.

But you are right that I replaced an initial intent with another use-case which didn't exist initially.

Let me fix that